<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFESPIMB040ScriptedAcquisition</class>
 <widget class="QWidget" name="QFESPIMB040ScriptedAcquisition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripted Acquisition</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btnNew">
       <property name="toolTip">
        <string>start new script</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="spimb040.qrc">
         <normaloff>:/spimb040/script_new.png</normaloff>:/spimb040/script_new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnOpen">
       <property name="toolTip">
        <string>load a script</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="spimb040.qrc">
         <normaloff>:/spimb040/script_open.png</normaloff>:/spimb040/script_open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSave">
       <property name="toolTip">
        <string>save current script</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="spimb040.qrc">
         <normaloff>:/spimb040/script_save.png</normaloff>:/spimb040/script_save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSyntaxCheck">
       <property name="toolTip">
        <string>perform syntax check for current script</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="spimb040.qrc">
         <normaloff>:/spimb040/script_syntaxcheck.png</normaloff>:/spimb040/script_syntaxcheck.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labScriptFilename">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="edtScript"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>350</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string> Script Help </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0,0">
        <item>
         <widget class="QLabel" name="labHelp">
          <property name="text">
           <string>---</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpenTemplate">
          <property name="text">
           <string>open template</string>
          </property>
          <property name="icon">
           <iconset resource="spimb040.qrc">
            <normaloff>:/spimb040/script_open.png</normaloff>:/spimb040/script_open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnOpenExample">
          <property name="text">
           <string>open example</string>
          </property>
          <property name="icon">
           <iconset resource="spimb040.qrc">
            <normaloff>:/spimb040/script_open.png</normaloff>:/spimb040/script_open.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QModernProgressWidget" name="progress" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labProgress">
            <property name="font">
             <font>
              <pointsize>7</pointsize>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="text">
        <string>&amp;Execute Script</string>
       </property>
       <property name="icon">
        <iconset resource="spimb040.qrc">
         <normaloff>:/spimb040/acquisitionsingle.png</normaloff>:/spimb040/acquisitionsingle.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QModernProgressWidget" name="widProgress" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel Script</string>
       </property>
       <property name="icon">
        <iconset resource="spimb040.qrc">
         <normaloff>:/spimb040/stage_error.png</normaloff>:/spimb040/stage_error.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QModernProgressWidget</class>
   <extends>QWidget</extends>
   <header>qmodernprogresswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="spimb040.qrc"/>
 </resources>
 <connections/>
</ui>
